fatiando/mesher.py

```diff
@@ -1720,3 +1720,433 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid (GeometricElement):
+
+    """
+    Create a triaxial ellipsoid.
+
+    .. note:: The coordinate system used is x -> North, y -> East
+    and z -> Down
+
+    Parameters:
+
+    * x, y, z : float
+        The coordinates of the center of the ellipsoid
+    * a, b, c : float
+        Semi-axes of the ellipsoid (a>b>c)
+    * alpha, gamma, delta
+        Orientation angles of the ellipsoid
+    * props : dict
+        Physical properties assigned to the ellipsoid.
+        Ex: ``props={'density':10,
+                     'remanence':[10000., 25., 40.],
+                     'k':[0.562, 0.485, 0.25, 90., 0., 0.]}``
+
+    Examples:
+
+        >>> e = TriaxialEllipsoid(1, 2, 3, 6, 5, 4, 10, 20, 30,{
+        ...                       'remanence':[10000., 25., 40.],
+        ...                       'k': [0.562, 0.485, 0.25, 90., 0., 0.]})
+        >>> e.props['remanence']
+        [10000.0, 25.0, 40.0]
+        >>> e.addprop('density', 20)
+        >>> print e.props['density']
```

Comment:
Avoid `print` statements in doctests. In Python 3, `print` is a function. You can use `print(e.props['density'])` in the doctests and it should work. This will make the transition to Python 3 easier.